pages/community/events/index.js

```diff
@@ -0,0 +1,206 @@
+/* eslint-disable react/no-unescaped-entities */
+import React, { useState } from 'react';
+import Button from '../../../components/buttons/Button';
+import GoogleCalendarButton from '../../../components/buttons/GoogleCalendarButton';
+import ICSFileButton from '../../../components/buttons/ICSFileButton';
+import Heading from '../../../components/typography/Heading';
+import { ArrowRightIcon } from '@heroicons/react/outline';
+import EventPostItem from '../../../components/navigation/EventPostItem';
+import EventFilter from '../../../components/navigation/EventFilter';
```

Comment:
I didn't understand why these components are part of `navigation`. You can create a folder named `community` if you have so many components only specified for Community sections.

Reply:
I don't think it's necessary to create a folder component called community. I believe it's best if it's in the navigation dir